server/swimos_agent/src/stores/map/mod.rs

```diff
@@ -362,7 +362,7 @@ where
     }
 }
 
-/// An [`EventHandler`] that will alter an entry in the map.
+///  An [event handler](crate::event_handler::EventHandler)`] that will alter an entry in the map.
```

Comment:
```suggestion
///  An [event handler](crate::event_handler::EventHandler)`] that may alter an entry in the map.
```
As the handler might not do anything